docs/src/add-services/varnish.md

```diff
@@ -146,8 +144,12 @@ This limitation may be lifted in the future.
 
 ## Stats endpoint
```

Comment:
:pick: Shouldn't that be an optional step of the flow? (Step 4)

Reply:
I didn't think so. It's a nice add on, but it's not really necessary to run Varnish itself. Plus it's complicated and involves adding another app. I wouldn't push it so strongly.